pytest_logger/plugin.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, config, logcfg):
         self._config = config
         self._logdirlinks = config.hook.pytest_logger_logdirlink(config=config)
         self._loggers = _loggers_from_logcfg(logcfg, config.getoption('log')) if logcfg._enabled else None
+        self._formatter = logcfg._formatter if isinstance(logcfg._formatter, logging.Formatter) else DefaultFormatter()
```

Comment:
Consider using "or" trick to pick given or default formatter like this `logcfg_.formatter or DefaultFormatter`.

Reply:
Changed in 666aa66